aten/src/ATen/native/README.md

```diff
@@ -179,6 +179,15 @@ There are a few situations where you might like to use this functionality:
 If you grep for `python_default_init`, you can find examples of this being used;
 in general, most functions will not need to use this.
 
+### `cpu_half`
+
+```
+cpu_half: True
+```
+
+By default, we do not generate dispatch for CPU half, because most kernels will
```

Comment:
nit: I'm not really a fan of comments that give me unsolicited advice (why don't I want to do this?  give me the facts!).  Honestly, I'd just not document this, but it's your call.

Reply:
I'll undocument it